src/state/content-settings/reducer.js

```diff
@@ -1,6 +1,14 @@
-import { SET_CONTENT_SETTINGS, SET_EDITOR_SETTINGS } from 'state/content-settings/types';
+import {
+  SET_CONTENT_SETTINGS,
+  SET_EDITOR_SETTINGS,
+  SET_CROP_RATIOS,
+} from 'state/content-settings/types';
 
-const reducer = (state = {}, action = {}) => {
+const initialState = {
```

Comment:
is it possible to only use `const initialState = {};` for this part?

Reply:
The default `cropRatio`, which is set to an empty array, is used in connected components. Another approach would be to have default parameters or default props in a component that uses it. But, it's actually a recommended approach to have an `initialState` as the default parameter of a reducer. That way, it also acts as the data structure definition for the entire state.